docs/conf.py

```diff
@@ -87,17 +87,25 @@
 # To configure AutoStructify
 def setup(app):
     from recommonmark.transform import AutoStructify
+    from m2r import MdInclude, convert
 
     params = {
         "auto_toc_tree_section": "Contents",
         "enable_eval_rst": True,
-        "enable_auto_doc_ref": True,
+        # "enable_auto_doc_ref": True,
         "enable_math": True,
         "enable_inline_math": True,
     }
     app.add_config_value("recommonmark_config", params, True)
     app.add_transform(AutoStructify)
 
+    # from m2r to make `mdinclude` work
+    app.add_config_value('no_underscore_emphasis', False, 'env')
+    app.add_config_value('m2r_parse_relative_links', False, 'env')
+    app.add_config_value('m2r_anonymous_references', False, 'env')
+    app.add_config_value('m2r_disable_inline_math', False, 'env')
+    app.add_directive('mdinclude', MdInclude)
+
```

Comment:
Do these changes also need to be added to the template itself? Meaning `AUTO_STRUCTIFY_CONF` in `extension.py`?

Reply:
We can add them to the template... it is a workaround and have some drawbacks... but they do allow usind `mdinclude`... which means that even files such as AUTHORS.md and CHANGELOG.md could be viable

Reply:
There is a recent [PR](https://github.com/sphinx-doc/sphinx/pull/7739) that if accepted would lift the need for a workaround...